app-games/src/main/java/com/aptoide/android/aptoidegames/home/BundlesView.kt

```diff
@@ -261,19 +260,20 @@ fun EmptyBundleView(height: Dp) {
 
 @Composable
 fun BundleHeader(
-  bundle: Bundle,
-  onClick: () -> Unit,
+  title: String,
+  icon: String?,
+  hasMoreAction: Boolean,
+  onClick: () -> Unit = {},
   titleColor: Color = Color.Unspecified,
-  actionColor: Color = AppTheme.colors.moreAppsViewBackColor,
+  iconColor: Color = primary,
```

Comment:
This is always the same, lets set it fixed inside `SeeMoreView` rather than pass it as default in arguments